digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/ProjectValidatorSpec.scala

```diff
@@ -1640,6 +1640,77 @@ class ProjectValidatorSpec extends FunSuite with Matchers {
       validationErrors3.foreach(e => e.validationError.value should be(projectValidator.ValidationErrorList.ErrorInValidationOfUnchangedLinks.value))
     }
   }
+   test("Test checkForInvalidUnchangedLinks When middle of road part is changed to another road number then there cant be unchanged after the transfer") {
```

Comment:
"unchanged after the transfer"
p.o.
"unchanged link after the transferred link"